pygfx/utils/load.py

```diff
@@ -562,3 +564,54 @@ def _volume_from_voxelgrid(vxl, cmap=None, clim="data"):
     vol.local.matrix = vxl.transform
 
     return vol
+
+
+def meshes_from_open3d(mesh_model, materials: bool = True) -> List[gfx.Mesh]:
+    """
+    Converts an Open3D TriangleMeshModel into a list of gfx.Mesh objects.
+
+    Parameters
+    ----------
+    mesh_model : open3d.visualization.rendering.TriangleMeshModel
+        The Open3D TriangleMeshModel to be converted.
+    materials : bool, optional
+        Whether to include materials in the conversion process. Default is True.
+
+    Returns
+    -------
+    results : list of gfx.Mesh
+        A list of gfx.Mesh objects converted from the input mesh model.
+
+    Raises
+    ------
+    ValueError
+        If the input is not an Open3D TriangleMeshModel.
+    """
+    import open3d
```

Comment:
Would be nice to apply the same `find_spec` trick as we do for `trimesh`, so that when open3d is not installed, ppl get a friendlier error message.
``py
if not find_spec("open3d"):
        raise ImportError(
            "The `open3d` library is required for this function: pip install open3d"
        )
```

Reply:
I have added the spec check for all methods that use open3d.